script.js

```diff
@@ -0,0 +1,63 @@
+//variables for form validation
+var form = document.getElementById("subscribe");
+var firstnameError = document.querySelector(".firstname-message");
+var surnameError = document.querySelector(".surname-message");
+var emailError = document.querySelector(".email-message");
+var commentError = document.querySelector(".comment-message");
+
+form.addEventListener("submit", function (e) {
+  e.preventDefault();
+  firstnameError.innerHTML = "";
+  surnameError.innerHTML = "";
+  emailError.innerHTML = "";
+  commentError.innerHTML = "";
+  submit.innerHTML = "";
+
+  var firstname = document.getElementById("firstname");
+  var surname = document.getElementById("surname");
+  var email = document.getElementById("email");
+  var comment = document.getElementById("comment");
+
+  var error = [];
+
+  // adding error messages
+  if (firstname.value.trim().length == 0) {
+    error.push({ firstname: "Please input your name" });
+  }
+
+  if (surname.value.trim().length == 0) {
+    error.push({ surname: "Please input your surname" });
+  }
+
+  if (
+    email.value.trim().length == 0 ||
+    email.value.trim().length != email.toLowerCase
+  ) {
+    error.push({ email: "Please input your email in the correct format" });
```

Comment:
- [ ] Good job implementing the form validation. It works as expected. Just a few minor tweaks and you're good to go.
- The email error message should show up close to the "submit" button.
- The submit button has some styling issues. Also, the form spacings are smaller than the design, kindly refer to the Figma design template and correct it.

_See images below for reference._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/123235673-f07c7700-d490-11eb-8932-e33b007287e9.png)) | ![image](https://user-images.githubusercontent.com/49621279/123235720-fc683900-d490-11eb-84be-5ed5a4e01369.png)